ui/pages/bridge/bridge.util.ts

```diff
@@ -159,3 +154,105 @@ export async function fetchBridgeTokens(
   });
   return transformedTokens;
 }
+
+// Returns a list of bridge tx quotes
+export async function fetchBridgeQuotes(
+  request: QuoteRequest,
+): Promise<QuoteResponse[]> {
+  const url = `${BRIDGE_API_BASE_URL}/getQuote?${Object.entries(request)
+    .map(([k, v]) => `${k}=${v}`)
+    .join('&')}`;
+  const quotes = await fetchWithCache({
+    url,
+    fetchOptions: { method: 'GET', headers: CLIENT_ID_HEADER },
+    cacheOptions: { cacheRefreshTime: 0 },
+    functionName: 'fetchBridgeQuotes',
+  });
+
+  const filteredQuotes = quotes.filter((quote: QuoteResponse) =>
```

Comment:
Would it be cleaner to use a nested validator for those object validation functions?